knative-operator/pkg/controller/knativekafka/knativekafka_controller.go

```diff
@@ -61,16 +61,22 @@ func newReconciler(mgr manager.Manager) (*ReconcileKnativeKafka, error) {
 		return nil, fmt.Errorf("failed to load KafkaChannel manifest: %w", err)
 	}
 
+	kafkaChannelPostInstallManifest, err := mf.ManifestFrom(mf.Path(os.Getenv("KAFKACHANNEL_POST_INSTALL_MANIFEST_PATH")))
+	if err != nil {
+		return nil, fmt.Errorf("failed to load KafkaChannel Post Install manifest: %w", err)
+	}
+
 	kafkaSourceManifest, err := mf.ManifestFrom(mf.Path(os.Getenv("KAFKASOURCE_MANIFEST_PATH")))
 	if err != nil {
 		return nil, fmt.Errorf("failed to load KafkaSource manifest: %w", err)
 	}
 
 	reconcileKnativeKafka := ReconcileKnativeKafka{
-		client:                  mgr.GetClient(),
-		scheme:                  mgr.GetScheme(),
-		rawKafkaChannelManifest: kafkaChannelManifest,
-		rawKafkaSourceManifest:  kafkaSourceManifest,
+		client:                             mgr.GetClient(),
+		scheme:                             mgr.GetScheme(),
+		rawKafkaChannelManifest:            kafkaChannelManifest,
+		rawKafkaChannelPostInstallManifest: kafkaChannelPostInstallManifest,
```

Comment:
Why do we need a specific post-install manifest? Could we not add it to the kafkaChannelManifest directly?

Reply:
You mean `.Append()` to the `kafkaChannelManifest`? I could, I guess - I was thinking to keep them separated :man_shrugging:

Reply:
I like the idea of a folder - Are you OK, I do refactor this afterwards?

(as a prep. for the 1.0 based post jobs for source/channel - and soon we have broker too - so channel makes sense)

Reply:
Personally, I would prefactor it, but I'm fine either way.